resource/ui/ts/models/proto.ts

```diff
@@ -37,6 +37,48 @@ module Models {
             last_update_nanos: number;
         }
 
+        /**
+         * Create a new object which implements MVCCStats interface, with zero
+         * values.
+         */
+        export function NewMVCCStats():MVCCStats {
```

Comment:
Maybe.  I've got a lot of forward work on the next part of this refactoring, i'm trying to break it up into understandable chunks; i'll add a ToDo in my current PR, if that's alright.